integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -133,27 +133,33 @@ public int chmod(String path, @mode_t long mode) {
   /**
    * Changes the owner of an Alluxio file.
    *
-   * This operation only works when the group mapping service is shelled based and the user is
-   * registered in unix. Otherwise it errors as not implemented. This is because the input uid and
-   * gid must match the user and group in Unix.
+   * This operation only works when the shell user group translation is enabled in Alluxio-Fuse,
+   * the user and group are registered in unix, and the user belongs to the group.
+   * Otherwise it errors as not implemented. This is because the input uid and gid
+   * must match the user and group in Unix and should be valid.
```

Comment:
what is the difference between matching a user and group in Unix and being valid?

Reply:
deleted, it is not needed since user/group does not need to have relationship.